pandas/_libs/missing.pyx

```diff
@@ -403,6 +404,9 @@ class NAType(C_NAType):
     def __repr__(self) -> str:
         return "<NA>"
 
+    def __float__(self):
+        return NAN
```

Comment:
Could probably also use `float("nan")` - not sure how numpy defines `np.nan`

Reply:
hey thanks for weighing in - do mean to return `float('nan')` instead? sure, but why?

Reply:
would avoid an import but `float("nan") is np.nan` fails so probably better to stick with `np.nan`